readme.md

```diff
@@ -18,321 +23,17 @@ npm install --save react-wooden-tree
-collapseIcon?: string;              // < The icon for collapsing parents.
-expandIcon?: string;                // < The icon for expanding parents.
-loadingIcon?: string;               // < The loading icon when loading data with ajax.
-errorIcon?: string;                 // < The icon displayed when lazyLoading went wrong.
-selectedIcon?: string;              // < The icon for selected nodes.
-
-// Styling
-changedCheckboxClass?: string;      // < Extra class for the changed checkbox nodes.
-selectedClass?: string;             // < Extra class for the selected nodes.
-
-// Callbacks
-/**
- * All changes made in the tree will be propagated upwards.
- * Every time the tree changes the node's data the callback will be fired.
- *
- * @param {string} id The node's id.
- * @param {string} dataType The currently changed information.
- * @param {boolean} newValue The newly assigned value.
- */
-onDataChange: (id: string, dataType: string, newValue: any) => void;
-
-/**
- * The function which will be called when a lazily loadable node is
- * expanded first time.
- *
- * @param {NodeProps} node The node of the node which has to be loaded.
- * @returns {Promise<NodeProps[]>} Promise about the children of the given node.
- */
-lazyLoad?: (node: NodeProps) => Promise<NodeProps[]>;
-```
-
-## Node option:
-For each node the following options can be defined (types from typescript are not obligatory):
-
-All options are optional expect the text.
-```typescript
-text: string;                       // < The text displayed in the node.
-nodes?: NodeProps[];                // < The children nodes.
-state?: {
-    expanded: boolean;
-    checked: boolean;
-    selected: boolean;
-    disabled: boolean;
-};                                  // < The node states: checked, selected, expanded, disabled
-
-checkable?: boolean;                // < If false the node is non checkable even if the checkbox is shown.
-hideCheckbox?: boolean;             // < If true then the checkbox is not shown - use whit showCheckbox option.
-
-selectable?: boolean;               // < Determines if the node can be selected.
-selectedIcon?: string;              // < Sets the selected icon for the node.
-
-lazyLoad?: boolean;                 // < Determines if the node calls the lazy loading function on expand.
-loading?: boolean;                  // < Determines if the node is currently loading: Null when error occurred
-
-attr?: {[key: string]: string};     // < The passed attributes which appear in HTML as well and are searchable.
-
-// Styling
-icon?: string;                      // < Custom icon for the node.
-iconColor?: string;                 // < The color of the custom icon.
-iconBackground?: string;            // < The background color of the custom icon.
-image?: string;                     // < Custom image for the node - preferred over the icon.
-classes?: string;                   // < Custom classes for the node.
-```
+To switch between the redux and non redux version in the 
```

Comment:
Instead of this I'd like to see both demos on the same page right after each other, with some header texts to explain which is which.